### ISSUE TITLE
trainer_config.yaml is outdated

### ISSUE THREAD

**Issue by Timotheeee** [Sebastian-Schuchmann/A.I.-from-Scratch---ML-Agents-Example#2]
there's a new format now:

behaviors:
  Jumper:
    trainer_type: ppo
    hyperparameters:
      batch_size: 128
      buffer_size: 2048
      learning_rate: 0.0003
      beta: 0.005
      epsilon: 0.2
      lambd: 0.95
      num_epoch: 3
      learning_rate_schedule: linear
    network_settings:
      normalize: false
      hidden_units: 64
      num_layers: 2
      vis_encode_type: simple
      memory: null
    reward_signals:
      extrinsic:
        gamma: 0.99
        strength: 1.0
    init_path: null
    keep_checkpoints: 5
    checkpoint_interval: 500000
    max_steps: 500000
    time_horizon: 64
    summary_freq: 1000
    threaded: true
    self_play: null
    behavioral_cloning: null
env_settings:
  env_path: null
  env_args: null
  base_port: 5005
  num_envs: 1
  seed: -1
engine_settings:
  width: 84
  height: 84
  quality_level: 5
  time_scale: 20
  target_frame_rate: -1
  capture_frame_rate: 60
  no_graphics: false
environment_parameters: null
checkpoint_settings:
  run_id: car10
  initialize_from: null
  load_model: false
  resume: false
  force: false
  train_model: false
  inference: false
debug: false

**Comment by Henenenerik** [comment]
When I paste this it still does not work, has the format changed again or has github dropped any formatting?
